src/layers/RouteLayer/RouteLayer.js

```diff
@@ -137,7 +137,7 @@ class RouteLayer extends CasaLayer {
 
     const via = viaPoints.map((v) => `!${v}`);
     const urlParams = {
-      key: this.apiKey || '',
+      [this.apiKeyName || 'key']: this.apiKey || '',
```

Comment:
Alternatively, we could defined the default value in the CasaLayer. Same for ZoneLayer.